code.js

```diff
@@ -1,3 +1,24 @@
 function divideAndConquerSum(a) {
-    return a;
+    if(a.length <= 1){
+        return a[0];
+    }
+
+    split = Math.floor((a.length-1)/3);
+    ans = 0;
+    
+    b = divideAndConquerSum(a.slice(0,split+1));
+    ans = ans + b[0];
+    if((a.length/3)-split > .5){
```

Comment:
Not sure if I understand what you're doing here. You could also check if the array has at least 3 elements before making recursive calls.

Reply:
sorry i acidently sent this when it wasnt done yet, sorry about that.